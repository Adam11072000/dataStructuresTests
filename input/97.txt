init 76
addImage 88
addImage 141
deleteImage 177
mergeSuperPixels 141 42 0
addImage 141
addImage 40
deleteImage 153
resetLabelScore 145 -5 212
deleteImage 88
getHighestScoredLabel 141 17
setLabelScore 25 4 144 31
addImage 40
addImage 141
deleteImage 48
quit