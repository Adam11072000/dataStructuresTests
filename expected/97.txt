init done.
setLabelScore: INVALID_INPUT
addImage: INVALID_INPUT
addImage: INVALID_INPUT
setLabelScore: INVALID_INPUT
addImage: SUCCESS
addImage: SUCCESS
mergeSuperPixels: INVALID_INPUT
addImage: SUCCESS
setLabelScore: INVALID_INPUT
mergeSuperPixels: SUCCESS
getHighestScoredLabel: FAILURE
setLabelScore: INVALID_INPUT
quit done.
